.travis.yml

```diff
@@ -11,10 +11,11 @@ install:
 - go get -u github.com/client9/misspell/cmd/misspell
 before_script:
 - $(exit $(go fmt ./... | wc -l))
-- go vet ./...
+- if [[ ! -f 'go_test.mod' ]]; then cp go.mod go_test.mod; cp go.sum go_test.sum; fi
+- go vet -modfile=go_test.mod ./...
 - find . -type f -name "*.go" | xargs misspell -error -locale US
 - staticcheck ./...
 script:
-- go test -i -race ./...
-- go test -v -run=TestNoRace -p=1 ./...
-- if [[ "$TRAVIS_GO_VERSION" =~ 1.16 ]]; then ./scripts/cov.sh TRAVIS; else go test -race -v -p=1 ./... --failfast; fi
+- go test -modfile=go_test.mod -i -race ./...
```

Comment:
Wonder if we not simply remove this line. It is even deprecated since go 1.16: https://travis-ci.com/github/nats-io/nats.go/jobs/495796803#L264

Reply:
Seems will be fine if we just remove, so took it out: https://github.com/golang/go/issues/41696